src/pages/ResultPage.jsx

```
@@ -257,24 +251,41 @@ function ResultPage() {
+          'https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/Duck/glTF/DuckCM.png',
+        scale: 0.1,
+        width_cm: product.dimensions?.width_cm || 50,
+        depth_cm: product.dimensions?.depth_cm || 50,
+        height_cm: product.dimensions?.height_cm || 50,
+      };
+
+      // 추가 테스트 모델 (다른 가구)
+      const additionalModelInfo = {
+        label: 'chair',
+        model_url:
+          'https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/Box/glTF-Binary/Box.glb',
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)